### ISSUE TITLE
Prevent repeated dialog toast messages

### ISSUE THREAD

**Issue by theronakpatel** [nextcloud/nextcloud-dialogs#753]
When there is an error, Nextcloud typically generates many repeated messages to the user.
We can at least modify the [nextcloud-dialogs](https://github.com/nextcloud/nextcloud-dialogs) library to not show a message already on screen.
Alternatively, identify a few cases (like storage full) and buffer those on the component generating the alert.

![Screenshot (4)](https://user-images.githubusercontent.com/11226784/215856801-978d2855-fab1-4947-b398-28f7156d974a.png)

**Comment by nickvergessen** [comment]
> Alternatively, identify a few cases (like storage full) and buffer those on the component generating the alert.

Yeah that should be done. I don't think it's the job of the lib to deduplicate, because it can not know if that is needed or not. Maybe the user pressed another button and again receives a "No permissions" error. So it should be triggered again.

Storage notification should therefore be fixed in the files app.

**Comment by theronakpatel** [comment]
@nickvergessen but cant we hide previous one first then show new notification!

**Comment by nickvergessen** [comment]
yeah the code that creates it can do that already.

**Comment by susnux** [comment]
It is already possible to hide a existing toast, we do this for example in the files app.
Closing as this should be handled on app level
